systemtests/tests/ndmp/etc/bareos/bareos-sd.d/device/FileStorage2.conf

```diff
@@ -8,4 +8,8 @@ Device {
   RemovableMedia = no;
   AlwaysOpen = no;
   Description = "File device. A connecting Director must have the same Name and MediaType."
+  Auto Inflate = in
+  Auto Deflate = out
```

Comment:
We should probably have a testcase with `Auto Deflate = out` and no `Auto Inflate` at all.
I guess we will also need to always force `Auto Inflate = in` for NDMP jobs, so NDMP-data is always decompressed if needed (and the autoxflate plugin is loaded).